test/link/table.txt

```diff
@@ -44,11 +44,11 @@ Elem:
 Custom:
  - name: "reloc.Elem"
   - section: Elem
-   - R_FUNC_INDEX_LEB   idx=0    addend=0    offset=0x6(file=0x38)
```

Comment:
Because we don't currently generate GLOBAL_ADDR relocations.  I haven't found a way to represent them in the wast format.  I guess we could use the binary builds to build them?